lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
Thanks for the detailed analysis @sarahlensing. So it looks like the main inconsistency is for *3. App is quit by user using recent apps menu swipe*

And then there is the issue of when the app is killed by the system which may not be the same as a user force quit honestly I am not sure.

Another nice way to get visibility into what services are running is under system settings *Developer options > Running services*